hugo.yaml

```diff
@@ -0,0 +1,119 @@
+baseURL: 'https://www.kubernetes.dev/'
+title: &title Kubernetes Contributors # Create a reference to use later
+theme:
+  - docsy
+enableRobotsTXT: true
+# This is a default
+#enableGitInfo: false
+contentDir: content/en
+
+# These are defaults
+#defaultContentLanguage: en
+#defaultContentLanguageInSubdir: false
+
+enableMissingTranslationPlaceholders: true
+disableKinds:
+  - taxonomy
+  - taxonomyTerm
+
+# Pygments is no longer used
+#pygmentsCodeFences: true
+#pygmentsUseClasses: false
+#pygmentsUseClassic: false
+#pygmentsStyle: tango
+
+permalinks:
+  blog: '/:section/:year/:month/:day/:slug/'
+
+imaging:
+  resampleFilter: CatmullRom
+# These are defaults
+#  quality: 75
+#  anchor: smart
+
+services:
+  googleAnalytics:
+    id: UA-00000000-0
+
+caches:
+  assets:
+    dir: ':cacheDir/_gen'
+    maxAge: -1
+  getcsv:
+    dir: ':cacheDir/:project'
+    maxAge: 60s
+  getjson:
+    dir: ':cacheDir/:project'
+    maxAge: 60s
+  images:
+    dir: ':cacheDir/_images'
+    maxAge: -1
+  modules:
+    dir: ':cacheDir/modules'
+    maxAge: -1
+
+languages:
+  en:
+    languageName: English
+    title: *title # Use the YAML reference instead of repeating
+
+markup:
+  goldmark:
+    renderer:
+      unsafe: true
+  highlight:
+    style: tango
+
+frontmatter:
+  date:
+    - date
+    - ':filename'
+    - publishDate
+    - lastmod
+
+params:
+  # This is used instead of the built-in top level `copyright` to utilize
+  # Docsy's partial which adds the copyright year automatically
+  copyright: The Kubernetes Authors
+#  These are defaults
+#  privacy_policy: ''
+#  version_menu: Releases
+#  archived_version: false
+#  version: '0.0'
+  gcs_engine_id: 8ad0f1b8442fece81
+#  This is deprecated and the new method doesn't need to be explicitly disabled
+#  algolia_docsearch: false
+#  TOffline search is disabled by default
+#  offlineSearch: false
+  ui:
+    sidebar_menu_compact: true
+#    Breadcrumbs are enabled by default
+#    breadcrumb_disable: false
+    sidebar_search_disable: false
+#    Navbar logo is enabled by default
+#    navbar_logo: true
+#    This is deprecated and footer about is hidden by default
+#    but enabling it is not required as we don't have an about page
+#    footer_about_disable: false
+    feedback:
+      enable: true
+  links:
+#   Updated icons to use latest syntax from FA.
+    user:
+      - name: Kubernetes Dev Mailing List
+        url: 'https://groups.google.com/a/kubernetes.io/group/dev'
+        icon: fa-solid fa-envelope
+        desc: Discussion and help from your fellow users
+      - name: Twitter
+        url: 'https://twitter.com/K8sContributors'
+        icon: fa-brands fa-x-twitter # Updated the bird icon to the X icon
```

Comment:
Omit this comment, but once you have, comment on this line in your own PR to call out the change.

Reply:
What about the other properties that I have commented in `hugo.yaml`. I have kept those properties commented on to get some feedback, should I remove those properties altogether or should I uncomment them.